instrumentation/spring/spring-batch-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/batch/v3_0/chunk/JavaConfigChunkRootSpanTest.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.chunk;
+
+import io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.runner.ApplicationConfigRunner;
+import io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.runner.SpringBatchApplication;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+class JavaConfigChunkRootSpanTest extends ChunkRootSpanTest {
+
+  @RegisterExtension
+  static final ApplicationConfigRunner runner =
+      new ApplicationConfigRunner(
+          () -> new AnnotationConfigApplicationContext(SpringBatchApplication.class));
```

Comment:
curious why you chose to use an extension instead of having it in `AbstractSpringBatchTest`?

Reply:
got it - this doesn't work, because it would require multiple inheritance

```java
class JavaConfigChunkRootSpanTest extends AbstractChunkRootSpanTest {

  @RegisterExtension
  static final ApplicationConfigRunner runner =
      new ApplicationConfigRunner(
          () -> new AnnotationConfigApplicationContext(SpringBatchApplication.class));
```

and 

```java
class JavaConfigBatchJobTest extends SpringBatchTest {

  @RegisterExtension
  static final ApplicationConfigRunner runner =
      new ApplicationConfigRunner(
          () -> new AnnotationConfigApplicationContext(SpringBatchApplication.class));

  public JavaConfigBatchJobTest() {
    super(runner);
  }
}
```